packages/fractal/fractal-transform-components/src/transform.test.js

```diff
@@ -108,14 +109,18 @@ describe('Component Transform', function () {
       expect(output).to.be.a('ComponentCollection');
       expect(output).to.have.property('length').that.equals(3);
     });
-    it('instantiates the component with config merged from the config files and the default components config', async function () {
-      const spy = sinon.spy(Component, 'from');
-      const fileCollection = FileCollection.from(items.slice(1));
-      const transform = componentTransform().transform;
-      const defaults = app.get('components.config.defaults');
-      await transform(fileCollection, {}, app);
-      expect(spy.args[1][0].config).to.eql(defaultsDeep({name: 'config.json', bar: 'baz', foo: 'bar'}, defaults));
-      spy.restore();
-    });
+    it('loads config via a loader instance');
+    it('passes the current file collection through when creating a new loader');
+    it('uses the first config file found when sorting all matches by filename');
+    it('correctly transforms nested components');
+    // it('instantiates the component with config merged from the config files and the default components config', async function () {
```

Comment:
How come this is taken out? Is it waiting on the VariantCollection changes mentioned?